modules/ROOT/pages/profiles-applications-overview.adoc

```diff
@@ -35,22 +35,12 @@ By default, it gives access to the Bonita User Application.
 
 By default, its gives access to Bonita Administrator Application. It includes:
```

Comment:
By default, its gives access to Bonita Administrator Application. It includes:  => By default, it gives access to Bonita Administrator Application, which includes: